### ISSUE TITLE
Interpreting data quality from sample-probs

### ISSUE THREAD

**Issue by kylepalos** [nanoporetech/modkit#266]
Hello,

Thank you for developing modkit and being so responsive to community support. 

I'm essentially wondering whether my direct RNA samples have any modifications that I can confidently infer besides m6A given the probability distributions below.

I have plant mRNA samples and matched IVT mRNA that I basecalled with the newest Dorado version (v0.8.0):

```
dorado basecaller -v hac,inosine_m6A,m5C,pseU --min-qscore 10 --emit-moves pod5/  --estimate-poly-a > v8_basecall/mRNA_all_mods.bam
```

I converted to fastq, mapped, and used modkit sample-probs (v 0.4.0):

```
modkit sample-probs --hist  mRNA_all_mods_mapped.bam

```

And I got the following distributions for m6A IVT and "raw" mRNA:
![Slide1](https://github.com/user-attachments/assets/f8d2f401-f140-427a-96c4-3b0fd70f3e6e)

Which I interpret as a lot of high probability m6A calls in the raw samples that are absent in the IVT sample - all good so far.

However, when I look at all the other mod types, e.g., Inosine, I see:
![Slide2](https://github.com/user-attachments/assets/ce452516-fdbb-4607-8200-4085f8818260)

Where the distribution basically looks identical b/w mRNA and IVT - this is the case for pseU and m5C as well. These mods are going to be present at substantially lower levels relative to m6A in mRNA but I still expect to see some.

Would you advise against examining modifications besides m6A from these data? Are there thresholding values that I can use to subtract out "noisy" IVT modification calls fom the "raw" samples? Would switching to the sup basecalling model be advisable in this case?

Thank you so much!
kyle

**Comment by kir1to455** [comment]
Hi, 
I used to get the same distributions when I was doing pseU before (V5.0.0) .

Actually, I have correlated the results of RNA004 pseU with BID-seq and PRAISE in HEK293T or Mouse Cell.
It only has 0.25 correlation. Unbelievable. This may be because RNA004 pseU does not consider U>C mutations.
https://github.com/nanoporetech/dorado/issues/867

Similarly, you can compare the results of RNA004 m5C with the experiment of identifying m5C method (m5C-TAC-seq or BS-seq).
I don't think the correlation between them will be too high.

In short, I believe Nanopore may not have optimized the models beyond m6A. If you have NSUN2 or NSUN6 knockdown( knock out) samples, try to compare the differences between the WT and knockdown.

Best wishes, 
Kirito

**Comment by kylepalos** [comment]
Hi @kir1to455,

Thanks for your response and suggestions. I will come back to this issue when I have supplemental experiments to compare the Nanopore results to.

Thanks again,
kyle
